index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hi there</title>
+  
+  <script type="text/javascript" src="js/jquery.js"></script>
+  
+   <script src="myscript.js" type="text/javascript"> 
+   
+  </script>
+  
+
+
+</head>
+<body>
+  This is a page
+  a simple page
+
+<div id="outputF">
```

Comment:
It is better to name variables with meaningfull name. For example `id='outputContainer'` instead of `id='outputF'`